deploy/helm/sumologic/conf/metrics/otelcol/pipeline.yaml

```diff
@@ -1,13 +1,22 @@
 exporters:
   - sumologic/default
+{{- if eq .Values.debug.metrics.metadata.print true }}
+  - debug
+{{- end }}
+{{- if eq (include "sumologic-mock.forward-metrics-metadata" .) "true" }}
+  - sumologic/sumologic-mock-default
+{{- end }}
 {{- if eq .Values.sumologic.metrics.sourceType "http" }}
-  - sumologic/apiserver
+  - sumologic/apiserverreceiver
   - sumologic/control_plane
```

Comment:
What's up with this change? I don't see the definition being changed anywhere.

Reply:
typo :see_no_evil: